lib/Dialect/TritonGPU/Transforms/AccelerateAMDMatmul.cpp

```diff
@@ -203,6 +203,9 @@ class BlockedToMFMA : public mlir::RewritePattern {
       return failure();
 
     auto CTALayout = ttg::getCTALayout(oldRetType.getEncoding());
```

Comment:
Why do we still need `CTALayout` here?

Reply:
I left this variable just in case. `oldRetType.getEncoding()` is a blocked layout (at least in cases I saw), and it still contains `CTALayout` field.

I left this part to check that CTALayout is "trivial", i.e. all dimensions are 1.
There are no other uses of this variable.

I do not want to remove this field from other layouts, because it will make IFUs harder.